recipe/meta.yaml

```diff
@@ -20,16 +20,16 @@ requirements:
     - python >=3.6
     - setuptools
   run:
-    - bw_migrations
+    - python >=3.6
     - bw2calc >=1.7.4
     - bw2data >=3.5.1
+    - bw_migrations
```

Comment:
...moved these packages to resemble the list in the [`setup.py` of `bw2io`](https://github.com/brightway-lca/brightway2-io/blob/0.8.10/setup.py):